src/lib.rs

```diff
@@ -572,13 +572,32 @@ impl TestKit {
         self.poll_events();
     }
 
+    /// Creates block with the given transactions.
+    /// Transactions that are in mempool will be ignored.
```

Comment:
What does "ignored" mean here? It seems that transactions that are already in the mempool will be added to the block. A more interesting question: what if some of transactions supplied to the method are already committed (i.e., present in one of previous blocks)?